contentcuration/contentcuration/viewsets/tree.py

```diff
@@ -27,7 +28,10 @@
 class TreeFilter(FilterSet):
```

Comment:
Should we inherit the new filter set here too?

Reply:
We currently enforce this inside the `list` method of the viewset, so I didn't add it here.

Reply:
Ah, we probably don't need to because we have a custom `list` method for the viewset right? It would fail anyway

Reply:
Yep